settings.gradle

```diff
@@ -23,7 +23,7 @@ buildCache {
   remote(com.github.burrunan.s3cache.AwsS3BuildCache) {
     region = 'us-west-2'
     bucket = 'opentelemetry-java-instrumentation-gradle-test1'
-    push = isCI
+    push = 'true'.equals(gradle.startParameter.projectProperties.get('pushBuildCache'))
```

Comment:
Why not `push = project.hasProperty('pushBuildCache')`?

Reply:
This is settings.gradle so there's no project here. But I changed to check env instead which I think is closer to what you had in mind in our chat conversation